packages/web3/src/api/index.ts

```diff
@@ -55,22 +55,6 @@ export class NodeProvider {
     this.miners = nodeApi.miners
     this.events = nodeApi.events
   }
-
-  // Have to duplicate the code above due to proxy pattern
-  reset(baseUrl: string, apiKey?: string): void {
-    const nodeApi = initializeNodeApi(baseUrl, apiKey)
-
-    this.wallets = nodeApi.wallets
```

Comment:
How about adding the `readonly` to these fields?

Reply:
Great one, done!